atest/testdata/tags/include_and_exclude.robot

```diff
@@ -25,3 +25,7 @@ Excl-12
 Excl-123
     [Tags]  excl_1  excl_2  excl_3
     No Operation
+
+Robot-exclude
+    [Tags]  robot:exclude  ROBOT:EXCLUDE
+    Log  this test will never be run
```

Comment:
- Could use `Fail`, not `Log` to make it more explicit this isn't run.
- Newline missing at the end of the file.